README.md

```diff
@@ -0,0 +1,69 @@
+# Islandora Paged Content PDF Batch [![Build Status](https://travis-ci.org/discoverygarden/islandora_paged_content_pdf_batch.png?branch=7.x)](https://travis-ci.org/discoverygarden/islandora_paged_content_pdf_batch)
+
+## Introduction
+
+This module implements a batch framework, as well as a basic ZIP ingester.
```

Comment:
This is copy pasta of copy pasta https://github.com/Islandora/islandora_batch#introduction we should document what this module does rather than what the module that provides the API it is using does.